test/e2e/util.go

```diff
@@ -2886,9 +2886,10 @@ func createGenericStatefulSetAW(context *context, name string) *arbv1.AppWrapper
 	return appwrapper
 }
 
-//NOTE: Recommend this test not to be the last test in the test suite it may pass
-//      may pass the local test but may cause controller to fail which is not
-//      part of this test's validation.
+// NOTE: Recommend this test not to be the last test in the test suite it may pass
+//
+//	may pass the local test but may cause controller to fail which is not
```

Comment:
Nit: Fix typo

Reply:
It seems the VS studio would want to automatically reformat that. Does this formating makes sense now ?